src/components/Forms/ContactFormFooter/ContactFormFooter.js

```diff
@@ -0,0 +1,96 @@
+import { useState } from 'react'
+import PropTypes from 'prop-types'
+import { useForm } from 'react-hook-form'
+import { FormAlert, onSubmitSuccess, onSubmitError } from '../'
+
+function ContactFormFooter() {
+  const [loading, setLoading] = useState(false)
+  const { register, handleSubmit, errors, reset } = useForm()
+
+  const onSubmit = async (formData, e) => {
```

Comment:
Please remove this file completely. Since we are using the contact form this should be no longer needed.